proto/api/v2/compliance_scan_schedule_service.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+
+option go_package = "v2";
+option java_package = "io.stackrox.proto.api.v2";
+
+import "google/protobuf/timestamp.proto";
+import weak "google/api/annotations.proto";
+import "api/v2/common.proto";
+import "api/v2/user.proto";
+
+package v2;
+
+// Represents the role of the node within the cluster
+enum NodeRole {
+    INFRA = 0;
+    WORKER = 1;
+    MASTER = 2;
+}
+
+// ClusterScanStatus holds status based on cluster in the event that a schedule was
+// successfully applied to some clusters but not others.
+// Next available tag: 3
+message ClusterScanStatus {
+    string                    cluster_id = 1;
+    repeated string           errors     = 2;
+}
+
+// Next available tag: 7
+message BaseComplianceScanScheduleSettings {
+    bool                  auto_apply_remediations  = 1;
+    bool                  auto_update_remediations = 2;
+    bool                  one_time_scan            = 3;
+    repeated string       profiles                 = 4;
+    Schedule              schedule                 = 5;
+    repeated NodeRole     node_roles               = 6;
+}
+
+// Next available tag: 4
+message ComplianceScanScheduleConfiguration {
+    string                                scan_name            = 1;
+    BaseComplianceScanScheduleSettings    scan_schedule_config = 2;
+    repeated string                       clusters             = 3;
+}
+
+// Next available tag: 7
+message ComplianceScanScheduleStatus {
+    string                                scan_name            = 1;
+    BaseComplianceScanScheduleSettings    scan_schedule_config = 2;
+    repeated ClusterScanStatus            clusters             = 3;
+    google.protobuf.Timestamp             created_time         = 4;
+    google.protobuf.Timestamp             last_updated_time    = 5;
+    // Most recent user to update the scan settings
+    SlimUser                              created_by           = 6;
+}
+
+// Next available tag: 2
+message ComplianceScanScheduleConfigRequest {
+    string scan_name = 1;
+}
+
+// Next available tag: 2
+message ListComplianceScanScheduleResponse {
+    repeated ComplianceScanScheduleStatus schedules = 1;
+}
+
+service ComplianceScanScheduleService {
+    // ListComplianceScanSchedules lists all the compliance operator schedules for the secured clusters
+    rpc ListComplianceScanSchedules (Empty) returns (ListComplianceScanScheduleResponse) {
+        option (google.api.http) = {
+            get: "/v2/compliance/scan/schedules"
```

Comment:
If we ever support one-time scans, the subpath `schedules` could become a misfit.